### ISSUE TITLE
On CentOS, the value returned by process.Percent()  is bigger than 100%

### ISSUE THREAD

**Issue by b3wsh** [shirou/gopsutil#599]
How I get cpu usage:
```
c.CPUUsage, err = p.Percent(time.Second * 1)
```
Centos Version: 6.4

**Comment by Lomanic** [comment]
Please edit your message with more information, such as example code where you can reproduce this issue (so we don't have to guess if you call `cpu.Percent` with percpu set to true or false, or which interval you use), which version of CentOS/Linux you are using.

`cpu.Percent` gets its data frop `/proc/stat`, please paste its content here.

**Comment by kconwayatlassian** [comment]
**Describe the bug**
Polling for CPU on linux sometimes reports over 100.00 for the float value when `false` for `percpu`.

**To Reproduce**
```go
start := time.Now()
stop := start.Add(time.Second)
for time.Now().Before(stop) {
    pctUsed, _ := cpu.Percent(time.Millisecond, false)
    fmt.Printf("%16.16f\n", pctUsed)
}
```

**Expected behavior**
0 and 100 are the lower and upper bounds, respectively.

**Environment (please complete the following information):**
 - [x] Linux: [paste contents of `/etc/os-release` and the result of `uname -a`]

uname -a
```
Linux COMPUTER-NAME 4.15.0-48-generic #51-Ubuntu SMP Wed Apr 3 08:28:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

/etc/os-relase
```
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

**Additional Context**
Sample output:
```
[49.9999999909050530]
[49.9999999886313162]
[50.0000000090949470]
[100.0000000500222086]
[50.0000000090949470]
[99.9999999772626325]
[50.0000000068212103]
[49.9999999909050530]
[100.0000000159161573]
[99.9999999818101060]
[99.9999999818101060]
[50.0000000068212103]
[99.9999999818101060]
[49.9999999909050530]
[99.9999999727151589]
[50.0000000090949470]
[99.9999999818101060]
[49.9999999886313162]
[50.0000000090949470]
[49.9999999909050530]
[50.0000000045474735]
[100.0000000500222086]
[100.0000000545696821]
[50.0000000068212103]
[100.0000000159161573]
[66.6666666530242367]
[66.6666666691930487]
[49.9999999909050530]
```

**Comment by Lomanic** [comment]
Thanks for this excellent reproduction case @kconwayatlassian, see new PR #707 with returned values as capped between 0 and 100.

**Comment by Lomanic** [comment]
Reopening as this not fixed in process package, hence why I put #707 in WIP.

**Comment by Lomanic** [comment]
I'm not even sure if this issue should be reopened in fact, as #776 reverted some of the fixes for this issue as it was breaking API, as reported in #755.
